gui/default/syncthing/core/syncthingController.js

```diff
@@ -1296,6 +1296,7 @@ angular.module('syncthing.core')
 
         $scope.addFolder = function () {
             $scope.currentFolder = {
+                path: $scope.config.options.parentFolder,
```

Comment:
Saw you post about nomenclature folder vs directory, thank you.
s/parentFolder/parentDirectory/

I tend to use $parentDirectory/$folderLabel for incoming shares.  With $folderID I would have no idea when using a file manager.  But for new shares... I guess a small bit of js to duplicate the 'Folder Label' into 'Folder Path' as it is being typed for the first time.  A bit more involved but a much better user experience.

I'll get back to you :)